tests/dialects/test_vertica.py

```diff
@@ -0,0 +1,88 @@
+from tests.dialects.test_dialect import Validator
+
+
+class TestVertica(Validator):
+    dialect = "vertica"  # type:ignore
+    maxDiff = None  # type:ignore
+
+    def test_vertica(self):
+        self.validate_all(
+            """
+            CREATE TABLE test_table (
+                col_tinyint INT64,
+            );
+            """,
+            write={
+                "vertica": """CREATE TABLE test_table (col_tinyint INT64)""",
+            },
+        )
+        self.validate_all(
+            """
+            CREATE TABLE test_table (
+                col_smallint INT64
+            );
+            """,
+            write={
+                "vertica": """CREATE TABLE test_table (col_smallint INT64)""",
+            },
+        )
```

Comment:
If the query/DDL you're testing remains the same after "roundtripping" i.e parsed and generated back in Vertica, you can also use `validate_identity` for simplicity. Also, no need to include the semicolon at the end of the query